{"level": "ERROR", "message": "Exception on /config_get [GET]", "timestamp": "2024-09-28T15:48:06.276998+00:00", "logger": "webapp", "module": "app", "function": "log_exception", "line": 838, "thread_name": "Thread-11 (process_request_thread)", "exc_info": "Traceback (most recent call last):\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 1473, in wsgi_app\n    response = self.full_dispatch_request()\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 882, in full_dispatch_request\n    rv = self.handle_user_exception(e)\n         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 880, in full_dispatch_request\n    rv = self.dispatch_request()\n         ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 865, in dispatch_request\n    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/webapp.py\", line 115, in get_config_get\n    return jsonify(response)\n           ^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/__init__.py\", line 170, in jsonify\n    return current_app.json.response(*args, **kwargs)  # type: ignore[return-value]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/provider.py\", line 214, in response\n    f\"{self.dumps(obj, **dump_args)}\\n\", mimetype=self.mimetype\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/provider.py\", line 179, in dumps\n    return json.dumps(obj, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/json/__init__.py\", line 238, in dumps\n    **kw).encode(obj)\n          ^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/json/encoder.py\", line 200, in encode\n    chunks = self.iterencode(o, _one_shot=True)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/json/encoder.py\", line 258, in iterencode\n    return _iterencode(o, 0)\n           ^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/provider.py\", line 121, in _default\n    raise TypeError(f\"Object of type {type(o).__name__} is not JSON serializable\")\nTypeError: Object of type Cursor is not JSON serializable"}
{"level": "ERROR", "message": "Exception on /config_get [GET]", "timestamp": "2024-09-28T15:49:15.434247+00:00", "logger": "webapp", "module": "app", "function": "log_exception", "line": 838, "thread_name": "Thread-13 (process_request_thread)", "exc_info": "Traceback (most recent call last):\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 1473, in wsgi_app\n    response = self.full_dispatch_request()\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 882, in full_dispatch_request\n    rv = self.handle_user_exception(e)\n         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 880, in full_dispatch_request\n    rv = self.dispatch_request()\n         ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 865, in dispatch_request\n    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/webapp.py\", line 115, in get_config_get\n    'printers': printer_list\n       ^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/__init__.py\", line 170, in jsonify\n    return current_app.json.response(*args, **kwargs)  # type: ignore[return-value]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/provider.py\", line 214, in response\n    f\"{self.dumps(obj, **dump_args)}\\n\", mimetype=self.mimetype\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/provider.py\", line 179, in dumps\n    return json.dumps(obj, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/json/__init__.py\", line 238, in dumps\n    **kw).encode(obj)\n          ^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/json/encoder.py\", line 200, in encode\n    chunks = self.iterencode(o, _one_shot=True)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/json/encoder.py\", line 258, in iterencode\n    return _iterencode(o, 0)\n           ^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/provider.py\", line 121, in _default\n    raise TypeError(f\"Object of type {type(o).__name__} is not JSON serializable\")\nTypeError: Object of type Cursor is not JSON serializable"}
{"level": "ERROR", "message": "Exception on /config_get [GET]", "timestamp": "2024-09-28T15:50:00.982182+00:00", "logger": "webapp", "module": "app", "function": "log_exception", "line": 838, "thread_name": "Thread-15 (process_request_thread)", "exc_info": "Traceback (most recent call last):\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 1473, in wsgi_app\n    response = self.full_dispatch_request()\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 882, in full_dispatch_request\n    rv = self.handle_user_exception(e)\n         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 880, in full_dispatch_request\n    rv = self.dispatch_request()\n         ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 865, in dispatch_request\n    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/webapp.py\", line 115, in get_config_get\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/__init__.py\", line 170, in jsonify\n    return current_app.json.response(*args, **kwargs)  # type: ignore[return-value]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/provider.py\", line 214, in response\n    f\"{self.dumps(obj, **dump_args)}\\n\", mimetype=self.mimetype\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/provider.py\", line 179, in dumps\n    return json.dumps(obj, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/json/__init__.py\", line 238, in dumps\n    **kw).encode(obj)\n          ^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/json/encoder.py\", line 200, in encode\n    chunks = self.iterencode(o, _one_shot=True)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/json/encoder.py\", line 258, in iterencode\n    return _iterencode(o, 0)\n           ^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/provider.py\", line 121, in _default\n    raise TypeError(f\"Object of type {type(o).__name__} is not JSON serializable\")\nTypeError: Object of type Cursor is not JSON serializable"}
{"level": "ERROR", "message": "Exception on /config_get [GET]", "timestamp": "2024-09-28T15:50:12.067472+00:00", "logger": "webapp", "module": "app", "function": "log_exception", "line": 838, "thread_name": "Thread-18 (process_request_thread)", "exc_info": "Traceback (most recent call last):\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 1473, in wsgi_app\n    response = self.full_dispatch_request()\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 882, in full_dispatch_request\n    rv = self.handle_user_exception(e)\n         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 880, in full_dispatch_request\n    rv = self.dispatch_request()\n         ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 865, in dispatch_request\n    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/webapp.py\", line 115, in get_config_get\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/__init__.py\", line 170, in jsonify\n    return current_app.json.response(*args, **kwargs)  # type: ignore[return-value]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/provider.py\", line 214, in response\n    f\"{self.dumps(obj, **dump_args)}\\n\", mimetype=self.mimetype\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/provider.py\", line 179, in dumps\n    return json.dumps(obj, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/json/__init__.py\", line 238, in dumps\n    **kw).encode(obj)\n          ^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/json/encoder.py\", line 200, in encode\n    chunks = self.iterencode(o, _one_shot=True)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/json/encoder.py\", line 258, in iterencode\n    return _iterencode(o, 0)\n           ^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/provider.py\", line 121, in _default\n    raise TypeError(f\"Object of type {type(o).__name__} is not JSON serializable\")\nTypeError: Object of type Cursor is not JSON serializable"}
{"level": "ERROR", "message": "Exception on /config_get [GET]", "timestamp": "2024-09-28T15:50:23.509715+00:00", "logger": "webapp", "module": "app", "function": "log_exception", "line": 838, "thread_name": "Thread-20 (process_request_thread)", "exc_info": "Traceback (most recent call last):\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 1473, in wsgi_app\n    response = self.full_dispatch_request()\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 882, in full_dispatch_request\n    rv = self.handle_user_exception(e)\n         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 880, in full_dispatch_request\n    rv = self.dispatch_request()\n         ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 865, in dispatch_request\n    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/webapp.py\", line 115, in get_config_get\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/__init__.py\", line 170, in jsonify\n    return current_app.json.response(*args, **kwargs)  # type: ignore[return-value]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/provider.py\", line 214, in response\n    f\"{self.dumps(obj, **dump_args)}\\n\", mimetype=self.mimetype\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/provider.py\", line 179, in dumps\n    return json.dumps(obj, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/json/__init__.py\", line 238, in dumps\n    **kw).encode(obj)\n          ^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/json/encoder.py\", line 200, in encode\n    chunks = self.iterencode(o, _one_shot=True)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/json/encoder.py\", line 258, in iterencode\n    return _iterencode(o, 0)\n           ^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/provider.py\", line 121, in _default\n    raise TypeError(f\"Object of type {type(o).__name__} is not JSON serializable\")\nTypeError: Object of type Cursor is not JSON serializable"}
{"level": "ERROR", "message": "Exception on /config_get [GET]", "timestamp": "2024-09-28T15:50:29.585109+00:00", "logger": "webapp", "module": "app", "function": "log_exception", "line": 838, "thread_name": "Thread-22 (process_request_thread)", "exc_info": "Traceback (most recent call last):\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 1473, in wsgi_app\n    response = self.full_dispatch_request()\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 882, in full_dispatch_request\n    rv = self.handle_user_exception(e)\n         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 880, in full_dispatch_request\n    rv = self.dispatch_request()\n         ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 865, in dispatch_request\n    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/webapp.py\", line 115, in get_config_get\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/__init__.py\", line 170, in jsonify\n    return current_app.json.response(*args, **kwargs)  # type: ignore[return-value]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/provider.py\", line 214, in response\n    f\"{self.dumps(obj, **dump_args)}\\n\", mimetype=self.mimetype\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/provider.py\", line 179, in dumps\n    return json.dumps(obj, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/json/__init__.py\", line 238, in dumps\n    **kw).encode(obj)\n          ^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/json/encoder.py\", line 200, in encode\n    chunks = self.iterencode(o, _one_shot=True)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/json/encoder.py\", line 258, in iterencode\n    return _iterencode(o, 0)\n           ^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/provider.py\", line 121, in _default\n    raise TypeError(f\"Object of type {type(o).__name__} is not JSON serializable\")\nTypeError: Object of type Cursor is not JSON serializable"}
{"level": "ERROR", "message": "Exception on /config_get [GET]", "timestamp": "2024-09-28T15:50:43.585714+00:00", "logger": "webapp", "module": "app", "function": "log_exception", "line": 838, "thread_name": "Thread-24 (process_request_thread)", "exc_info": "Traceback (most recent call last):\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 1473, in wsgi_app\n    response = self.full_dispatch_request()\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 882, in full_dispatch_request\n    rv = self.handle_user_exception(e)\n         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 880, in full_dispatch_request\n    rv = self.dispatch_request()\n         ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 865, in dispatch_request\n    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/webapp.py\", line 115, in get_config_get\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/__init__.py\", line 170, in jsonify\n    return current_app.json.response(*args, **kwargs)  # type: ignore[return-value]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/provider.py\", line 214, in response\n    f\"{self.dumps(obj, **dump_args)}\\n\", mimetype=self.mimetype\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/provider.py\", line 179, in dumps\n    return json.dumps(obj, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/json/__init__.py\", line 238, in dumps\n    **kw).encode(obj)\n          ^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/json/encoder.py\", line 200, in encode\n    chunks = self.iterencode(o, _one_shot=True)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/json/encoder.py\", line 258, in iterencode\n    return _iterencode(o, 0)\n           ^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/provider.py\", line 121, in _default\n    raise TypeError(f\"Object of type {type(o).__name__} is not JSON serializable\")\nTypeError: Object of type Cursor is not JSON serializable"}
{"level": "ERROR", "message": "Exception on /config_get [GET]", "timestamp": "2024-09-28T15:50:47.823443+00:00", "logger": "webapp", "module": "app", "function": "log_exception", "line": 838, "thread_name": "Thread-26 (process_request_thread)", "exc_info": "Traceback (most recent call last):\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 1473, in wsgi_app\n    response = self.full_dispatch_request()\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 882, in full_dispatch_request\n    rv = self.handle_user_exception(e)\n         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 880, in full_dispatch_request\n    rv = self.dispatch_request()\n         ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 865, in dispatch_request\n    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/webapp.py\", line 115, in get_config_get\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/__init__.py\", line 170, in jsonify\n    return current_app.json.response(*args, **kwargs)  # type: ignore[return-value]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/provider.py\", line 214, in response\n    f\"{self.dumps(obj, **dump_args)}\\n\", mimetype=self.mimetype\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/provider.py\", line 179, in dumps\n    return json.dumps(obj, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/json/__init__.py\", line 238, in dumps\n    **kw).encode(obj)\n          ^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/json/encoder.py\", line 200, in encode\n    chunks = self.iterencode(o, _one_shot=True)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/json/encoder.py\", line 258, in iterencode\n    return _iterencode(o, 0)\n           ^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/json/provider.py\", line 121, in _default\n    raise TypeError(f\"Object of type {type(o).__name__} is not JSON serializable\")\nTypeError: Object of type Cursor is not JSON serializable"}
{"level": "ERROR", "message": "Exception on /config/printers/add [POST]", "timestamp": "2024-09-28T16:02:19.957768+00:00", "logger": "webapp", "module": "app", "function": "log_exception", "line": 838, "thread_name": "Thread-7 (process_request_thread)", "exc_info": "Traceback (most recent call last):\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 1473, in wsgi_app\n    response = self.full_dispatch_request()\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 882, in full_dispatch_request\n    rv = self.handle_user_exception(e)\n         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 880, in full_dispatch_request\n    rv = self.dispatch_request()\n         ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 865, in dispatch_request\n    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask_httpauth.py\", line 174, in decorated\n    return self.ensure_sync(f)(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/webapp.py\", line 154, in add_printer\n    return jsonify({\"status\": \"error\", \"message\": \"Access code already in use\"})\n               ^^^^^^^^^^^^^^^^^^^^^^^\nNameError: name 'printer_list_collection' is not defined"}
{"level": "WARNING", "message": "unable to connect to printer - reason: timed out", "timestamp": "2024-09-28T16:20:32.202429+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "start_session", "line": 238, "thread_name": "Thread-5 (monitor_printer)", "exception": "Traceback (most recent call last):\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/bpm/bambuprinter.py\", line 235, in start_session\n    self.client.connect(self.config.hostname, self.config.mqtt_port, 60)\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/paho/mqtt/client.py\", line 1435, in connect\n    return self.reconnect()\n           ^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/paho/mqtt/client.py\", line 1598, in reconnect\n    self._sock = self._create_socket()\n                 ^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/paho/mqtt/client.py\", line 4609, in _create_socket\n    sock = self._create_socket_connection()\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/paho/mqtt/client.py\", line 4640, in _create_socket_connection\n    return socket.create_connection(addr, timeout=self._connect_timeout, source_address=source)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/socket.py\", line 852, in create_connection\n    raise exceptions[0]\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/socket.py\", line 837, in create_connection\n    sock.connect(sa)\nTimeoutError: timed out\n"}
{"level": "WARNING", "message": "unable to connect to printer - reason: timed out", "timestamp": "2024-09-28T19:53:36.906753+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "start_session", "line": 238, "thread_name": "Thread-6 (monitor_printer)", "exception": "Traceback (most recent call last):\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/bpm/bambuprinter.py\", line 235, in start_session\n    self.client.connect(self.config.hostname, self.config.mqtt_port, 60)\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/paho/mqtt/client.py\", line 1435, in connect\n    return self.reconnect()\n           ^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/paho/mqtt/client.py\", line 1598, in reconnect\n    self._sock = self._create_socket()\n                 ^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/paho/mqtt/client.py\", line 4609, in _create_socket\n    sock = self._create_socket_connection()\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/paho/mqtt/client.py\", line 4640, in _create_socket_connection\n    return socket.create_connection(addr, timeout=self._connect_timeout, source_address=source)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/socket.py\", line 852, in create_connection\n    raise exceptions[0]\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/socket.py\", line 837, in create_connection\n    sock.connect(sa)\nTimeoutError: timed out\n"}
{"level": "WARNING", "message": "unable to connect to printer - reason: timed out", "timestamp": "2024-09-28T19:53:36.907215+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "start_session", "line": 238, "thread_name": "Thread-3 (monitor_printer)", "exception": "Traceback (most recent call last):\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/bpm/bambuprinter.py\", line 235, in start_session\n    self.client.connect(self.config.hostname, self.config.mqtt_port, 60)\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/paho/mqtt/client.py\", line 1435, in connect\n    return self.reconnect()\n           ^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/paho/mqtt/client.py\", line 1598, in reconnect\n    self._sock = self._create_socket()\n                 ^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/paho/mqtt/client.py\", line 4609, in _create_socket\n    sock = self._create_socket_connection()\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/paho/mqtt/client.py\", line 4640, in _create_socket_connection\n    return socket.create_connection(addr, timeout=self._connect_timeout, source_address=source)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/socket.py\", line 852, in create_connection\n    raise exceptions[0]\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/socket.py\", line 837, in create_connection\n    sock.connect(sa)\nTimeoutError: timed out\n"}
{"level": "WARNING", "message": "unable to connect to printer - reason: timed out", "timestamp": "2024-09-28T19:53:36.907582+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "start_session", "line": 238, "thread_name": "Thread-4 (monitor_printer)", "exception": "Traceback (most recent call last):\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/bpm/bambuprinter.py\", line 235, in start_session\n    self.client.connect(self.config.hostname, self.config.mqtt_port, 60)\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/paho/mqtt/client.py\", line 1435, in connect\n    return self.reconnect()\n           ^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/paho/mqtt/client.py\", line 1598, in reconnect\n    self._sock = self._create_socket()\n                 ^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/paho/mqtt/client.py\", line 4609, in _create_socket\n    sock = self._create_socket_connection()\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/paho/mqtt/client.py\", line 4640, in _create_socket_connection\n    return socket.create_connection(addr, timeout=self._connect_timeout, source_address=source)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/socket.py\", line 852, in create_connection\n    raise exceptions[0]\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/socket.py\", line 837, in create_connection\n    sock.connect(sa)\nTimeoutError: timed out\n"}
{"level": "WARNING", "message": "unable to connect to printer - reason: timed out", "timestamp": "2024-09-28T19:53:36.907774+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "start_session", "line": 238, "thread_name": "Thread-2 (monitor_printer)", "exception": "Traceback (most recent call last):\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/bpm/bambuprinter.py\", line 235, in start_session\n    self.client.connect(self.config.hostname, self.config.mqtt_port, 60)\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/paho/mqtt/client.py\", line 1435, in connect\n    return self.reconnect()\n           ^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/paho/mqtt/client.py\", line 1598, in reconnect\n    self._sock = self._create_socket()\n                 ^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/paho/mqtt/client.py\", line 4609, in _create_socket\n    sock = self._create_socket_connection()\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/paho/mqtt/client.py\", line 4640, in _create_socket_connection\n    return socket.create_connection(addr, timeout=self._connect_timeout, source_address=source)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/socket.py\", line 852, in create_connection\n    raise exceptions[0]\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/socket.py\", line 837, in create_connection\n    sock.connect(sa)\nTimeoutError: timed out\n"}
{"level": "WARNING", "message": "unable to connect to printer - reason: timed out", "timestamp": "2024-09-28T19:53:36.907943+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "start_session", "line": 238, "thread_name": "Thread-1 (monitor_printer)", "exception": "Traceback (most recent call last):\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/bpm/bambuprinter.py\", line 235, in start_session\n    self.client.connect(self.config.hostname, self.config.mqtt_port, 60)\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/paho/mqtt/client.py\", line 1435, in connect\n    return self.reconnect()\n           ^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/paho/mqtt/client.py\", line 1598, in reconnect\n    self._sock = self._create_socket()\n                 ^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/paho/mqtt/client.py\", line 4609, in _create_socket\n    sock = self._create_socket_connection()\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/paho/mqtt/client.py\", line 4640, in _create_socket_connection\n    return socket.create_connection(addr, timeout=self._connect_timeout, source_address=source)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/socket.py\", line 852, in create_connection\n    raise exceptions[0]\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/socket.py\", line 837, in create_connection\n    sock.connect(sa)\nTimeoutError: timed out\n"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-09-28T19:57:03.921099+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-09-28T19:57:03.924096+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-09-28T19:57:03.924743+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-09-28T19:57:04.080065+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-09-28T19:57:04.081517+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-09-28T19:57:04.084941+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-09-28T20:00:19.096864+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-09-28T20:00:19.096863+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-09-28T20:00:19.096855+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-09-28T20:04:22.856304+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-09-28T20:04:22.856301+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-09-28T20:04:22.859008+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-09-28T20:04:22.944210+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-09-28T20:04:22.948353+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-09-28T20:04:22.948351+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-09-28T20:18:49.836828+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-09-28T20:18:49.975982+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-09-28T20:18:51.782938+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-09-28T20:18:51.876754+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-09-28T20:19:00.529987+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-09-28T20:19:30.114219+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "ERROR", "message": "Exception on /shutdown [POST]", "timestamp": "2024-09-28T20:20:51.232615+00:00", "logger": "mgmtapp", "module": "app", "function": "log_exception", "line": 838, "thread_name": "Thread-57 (process_request_thread)", "exc_info": "Traceback (most recent call last):\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 1473, in wsgi_app\n    response = self.full_dispatch_request()\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 882, in full_dispatch_request\n    rv = self.handle_user_exception(e)\n         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask_cors/extension.py\", line 194, in wrapped_function\n    return cors_after_request(app.make_response(f(*args, **kwargs)))\n                                                ^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 880, in full_dispatch_request\n    rv = self.dispatch_request()\n         ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 865, in dispatch_request\n    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/mgmtapp.py\", line 284, in shutdown\n    shutdown_server()\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/mgmtapp.py\", line 295, in shutdown_server\n    raise RuntimeError('Not running with the Werkzeug Server')\nRuntimeError: Not running with the Werkzeug Server"}
{"level": "ERROR", "message": "Exception on /shutdown [POST]", "timestamp": "2024-09-28T20:22:48.212804+00:00", "logger": "mgmtapp", "module": "app", "function": "log_exception", "line": 838, "thread_name": "Thread-19 (process_request_thread)", "exc_info": "Traceback (most recent call last):\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 1473, in wsgi_app\n    response = self.full_dispatch_request()\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 882, in full_dispatch_request\n    rv = self.handle_user_exception(e)\n         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask_cors/extension.py\", line 194, in wrapped_function\n    return cors_after_request(app.make_response(f(*args, **kwargs)))\n                                                ^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 880, in full_dispatch_request\n    rv = self.dispatch_request()\n         ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/venv/lib/python3.12/site-packages/flask/app.py\", line 865, in dispatch_request\n    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/mgmtapp.py\", line 284, in shutdown\n    global printer_status\n    ^^^^^^^^^^^^^^^^^\n  File \"/Users/ryanvogel/Local Laptop Project/bambu-dashboard/mgmtapp.py\", line 295, in shutdown_server\n    for printer in printers:\n        ^^^^^^^^^^^^^^^^^^^^^\nRuntimeError: Not running with the Werkzeug Server"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-10-04T19:00:12.600311+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-10-04T19:00:12.744360+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-10-04T19:00:15.906159+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-10-04T19:00:16.023910+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-10-04T19:00:17.439120+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-10-04T19:01:02.199593+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-10-04T19:01:20.716566+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-10-04T19:01:29.154342+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-10-04T19:01:29.736832+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
{"level": "WARNING", "message": "unknown message type received", "timestamp": "2024-10-04T19:01:29.838261+00:00", "logger": "bambuprinter", "module": "bambuprinter", "function": "_on_message", "line": 915, "thread_name": "bambuprinter-session"}
